package.json

```diff
@@ -4,33 +4,33 @@
   "version": "0.4.0",
   "type": "module",
   "scripts": {
-    "dev": "ROTO_API_HOST=https://rest.bgp-api.net ROUTINATOR_API_HOST=https://routinator.do.nlnetlabs.nl vite",
+    "dev": "ROTO_API_HOST=https://rest.bgp-api.net vite",
```

Comment:
Shouldn’t dev still have have `ROUTINATOR_API_HOST=https://routinator.nlnetlabs.nl`?

Reply:
The problem is that CORS headers will prevent the API from being loaded in development. I've added a server proxy configuration te circumvent this when running `yarn dev`.

Reply:
I’m trying to remember: Who would need to set the CORS headers to allow the link? routinator.nlnetlabs.nl, right?

Reply:
The CORS header (`Access-Control-Allow-Origin`) would be necessary on the API endpoint to allow browsers to load it from any other origin than routinator.nlnetlabs.nl.

Reply:
Hm, actually, the server proxy configuration is quite neat, so let’s just stick to that.